lite/tests/detection/test_average_precision.py

```diff
@@ -623,161 +631,175 @@ def test_ap_false_negatives_two_datums_one_only_with_different_class_high_confid
         assert m in actual_metrics
 
 
-def test_ap_ranked_pair_ordering(detection_ranked_pair_ordering: Detection):
-
-    loader = DataLoader()
-    loader.add_data(detections=[detection_ranked_pair_ordering])
-    evaluator = loader.finalize()
-
-    assert evaluator.metadata == {
-        "ignored_prediction_labels": [
-            ("class", "label4"),
-        ],
-        "missing_prediction_labels": [],
-        "n_datums": 1,
-        "n_groundtruths": 3,
-        "n_labels": 4,
-        "n_predictions": 4,
-    }
+def test_ap_ranked_pair_ordering(
+    detection_ranked_pair_ordering: Detection,
+    detection_ranked_pair_ordering_with_bitmasks: Detection,
+    detection_ranked_pair_ordering_with_polygons: Detection,
+):
+    for input_, method in [
```

Comment:
all of these tests are the same as before, I'm just iterating through multiple inputs and methods (e.g., `add_bounding_boxes`) to make sure they all return the same output